emails/grunt/aliases.yaml

```diff
@@ -1,5 +1,4 @@
 default:
-  - 'clean'
```

Comment:
This is now handled by `make clean` since we need to clean the folder before grunt is even invoked